server/db/models/products.js

```diff
@@ -0,0 +1,44 @@
+//define Products model here
+const Sequelize = require('sequelize');
+const { db } = require('./../database.js');
+
+const { UUID, UUIDV4, STRING, DECIMAL, TEXT, INTEGER } = Sequelize;
+
+const Products = db.define('products', {
+    productId: {
+        primaryKey: true,
+        type: UUID,
+        defaultValue: UUIDV4,
+    },
+    productName: {
+        type: STRING,
+        allowNull: false,
+        validate: {
+            notEmpty: {
+                args: true,
+                msg: 'Product name cannot be empty',
+            },
+            notNull: {
+                arg: true,
+                msg: 'Product name cannot be null',
+            },
+        },
+    },
+    productDescription: {
+        type: TEXT,
+    },
+    unitPrice: {
+        type: DECIMAL(10, 2),
+        allowNull: false,
+    },
+    inventory: {
+        type: INTEGER,
+        allowNull: false,
+    },
+    productImage: {
+        type: STRING,
+        defaultValue: 'No image'
```

Comment:
Should we initialize to an actual image source? just something from google images.

Reply:
Fair point. But I think we can just handle non-images with logic on the client. I'll seed the db with images from google images.

Reply:
Yeah, I just wanted to fill the column in case a product doesn't have an image.